README.md

```diff
@@ -6,21 +6,25 @@
 The Service Mesh Interface (SMI) is a specification for service meshes that run
 on Kubernetes. It defines a common standard that can be implemented by a variety
 of providers. This allows for both standardization for end-users and innovation
-by providers of Service Mesh Technology. It enables flexibility and
-interoperability.
-
-This specification consists of multiple APIs:
-
-* [Traffic Access Control](traffic-access-control.md) - configure access to specific
-  pods and routes based on the identity of a client for locking down
+by providers of Service Mesh Technology. SMI enables flexibility and
+interoperability, and covers the most common service mesh capabilities
+implemented by the following APIs:
+
+* Traffic policy – apply policies like identity and transport encryption
+  across services
+  * [Traffic Access Control](traffic-access-control.md) - configure access to
+  specific pods and routes based on the identity of a client for locking down
   applications to only allowed users and services.
-* [Traffic Specs](traffic-specs.md) - define how traffic looks on a per-protocol
-  basis. These resources work in concert with access control and other types of
-  policy to manage traffic at a protocol level.
-* [Traffic Split](traffic-split.md) - incrementally direct percentages of
+  * [Traffic Specs](traffic-specs.md) - define how traffic looks on a
```

Comment:
Hmm, traffic specs are being used in management as well. I think it fits here for now, but feels wrong in the long term. Should there be a `Traffic configuration` section?

Reply:
that's a good point

Reply:
traffic specs are required on traffic target where as it is optional on traffic split. Does that make it okay to leave under policy for now?

Reply:
Well, I'm trying to reconcile what's on the website:
- Traffic policy – apply policies like identity and transport encryption across services
- Traffic telemetry – capture key metrics like error rate and latency between services
- Traffic management – shift traffic between different services

with what's listed as available APIs:
- Traffic Access Control - configure access to specific pods and routes based on the identity of a client for locking down applications to only allowed users and services.
- Traffic Specs - define how traffic looks on a per-protocol basis. These resources work in concert with access control and other types of policy to manage traffic at a protocol level.
- Traffic Split - incrementally direct percentages of traffic between various services to assist in building out canary rollouts.
- Traffic Metrics - expose common traffic metrics for use by tools such as dashboards and autoscalers.

@grampelberg are you suggesting that the three categories on the website are not sufficient as umbrella categories? Or is it okay to have an API in more than one category, in which case a matrix is needed instead of splitting them as I am?

Reply:
Traffic specs don't actually do anything (unlike the other three), they just add configuration. To @michelleN's point they're optional for splits. I'm not sure what the best solution is, but I expect there'll be more APIs that fit into the configuration bucket in the future.

FWIW, most of this is future looking and it'd be okay to leave it as you've got it now. We'll just want to revisit in the future as the spec evolves.

Reply:
@grampelberg Thank you. If you think this is usable as-is, and will add more clarity than it adds confusion, I'd appreciate you approving this. Or I'd be equally happy with you re-wording it! =)

Reply:
Oh yeah! I thought I'd done that already =)